### PR TITLE
add ABI tab

### PR DESCRIPTION
**Pull request by nrryuya**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -206,13 +216,15 @@ class App extends Component {
...
+        <Menu tabular widths={5}>
           <Menu.Item active={activeItem == 'bytecode'} name="bytecode" onClick={this.onClickTab}>bytecode</Menu.Item>
           <Menu.Item active={activeItem == 'bytecode_runtime'} name="bytecode_runtime" onClick={this.onClickTab}>runtime bytecode</Menu.Item>
+          <Menu.Item active={activeItem == 'abi'} name="abi" onClick={this.onClickTab}>abi</Menu.Item>
           <Menu.Item active={activeItem == 'ir'} name="ir" onClick={this.onClickTab}>LLL</Menu.Item>
+          <Menu.Item active={activeItem == 'source_map_asm_list'} name="source_map_asm_list" onClick={this.onClickTab}>asm</Menu.Item>
         </Menu>
         <Segment attached='bottom'>
-          {(activeItem == 'ir') ? this.renderLLL(message) : this.renderBytecode(message)}
+          {(['ir', 'source_map_asm_list'].indexOf(activeItem) + 1) ? this.renderText(message) : this.renderBytecode(message)}
```
Comment (yudetamago) [inline src/App.js]: It seems that applying `renderText` instead of `renderBytecode` to abi is better for readability.

File: src/App.js [hunk 2]
```diff
@@ -159,7 +161,9 @@ class App extends Component {
       return {
         bytecode: this.state.compilationResult['bytecode'],
         bytecode_runtime: this.state.compilationResult['bytecode_runtime'],
-        ir: this.state.compilationResult['ir']
+        abi: JSON.stringify(this.state.compilationResult['abi']),
+        ir: this.state.compilationResult['ir'],
+        source_map_asm_list: this.state.compilationResult['source_map_asm_list']
```
Comment (yudetamago) [inline src/App.js]: How to get `source_map_asm_list` ?
It seems that current version of vyper.live doesn't return it...

Comment (nrryuya) [reply src/App.js]: I misunderstood that `vyper-serve` has already got to return that. I'll remove this.

File: src/App.js (hunk 1 above)
Comment (nrryuya) [inline src/App.js]: `['ir', 'source_map_asm_list'].indexOf(activeItem) + 1) ?` 

Help me !!!

Comment (nrryuya) [reply src/App.js]: OK. I'll try that with JSON formatter.

Comment (yudetamago) [reply src/App.js]: Maybe
```javascript
['ir', 'source_map_asm_list'].indexOf(activeItem) >= 0
```
 is what you want 👍

Comment (yudetamago) [reply src/App.js]: The simplest solution is pass number of spaces for indent to `JSON.stringify` like this.
```javascript
JSON.stringify(this.state.compilationResult['abi'], null, 4)
```

**Pull request merged**